sources/advertisements/VRM New Core/Controllers/FetchVRMItemController.swift

```diff
@@ -0,0 +1,41 @@
+//  Copyright 2018, Oath Inc.
+//  Licensed under the terms of the MIT License. See LICENSE.md file in project root for terms.
+
+import Foundation
+import PlayerCore
+
+final class FetchVRMItemController {
+    
+    typealias VASTXMLFetcher = (URL) -> Future<Result<String>>
```

Comment:
do we need this typealias - it makes me confused - the signature is pretty self explanatory

Reply:
dropped